libnmstate/netapplier.py

```diff
@@ -65,7 +66,10 @@ def apply(
     :rtype: str
     """
     logging.debug(f"Nmstate version: {get_version()}")
-    logging.debug(f"Applying desire state: {desired_state}")
+
+    desired_state_without_secrets = copy.deepcopy(desired_state)
+    hide_the_secrets(desired_state_without_secrets)
+    logging.debug(f"Applying desire state: {desired_state_without_secrets}")
```

Comment:
This `desired_state_without_secrets` is not being used.

Reply:
It is used by logging.

Reply:
Oh you are right. My bad.